hermetic_build/library_generation/owlbot/templates/java_library/.kokoro/build.sh

```diff
@@ -1,12 +1,11 @@
 #!/bin/bash
-
-# Copyright 2018 Google LLC
+# Copyright 2019 Google LLC
```

Comment:
Is this copyright year change and the https change below intentional?

Reply:
This file is based on a copy from java-storage. I fixed it to show year 2025 and using `https`

Reply:
Thanks for pointing it out.